src/components/Book/BookAbout/BookAbout.tsx

```diff
@@ -28,13 +60,27 @@ const BookAbout: React.FC = () => {
       <div className={classes['c-book-details__about']}>
         <h1>{book.title}</h1>
         <div className={classes['c-book-details__author']}>{book.author}</div>
-        <div className={classes['c-book-details__description']}>{book.description}</div>
+        <div className={classes['c-book-details__description']}>About book: {book.description}</div>
         <div className={classes['c-book-details__copies']}>
           Number of copies : {book.numOfCopies}
         </div>
+        <div className={classes['c-book-details__days']}>
+          <div style={{ marginBottom: '0.5rem' }}> {`Days (Renting will start from today):`}</div>
+          <Input
+            type={'number'}
+            placeholder={'days...'}
+            id={'days'}
+            name={'days'}
+            setValue={changeDaysHandler}
+          />
+        </div>
         <div className={classes['c-book-details__buttons']}>
-          <Button name={'RENT'} />
+          <Button name={'RENT'} clickHandler={rentBookHandler} />
+          {isAdministrator() ? (
+            <Button name={'CREATE COPY'} clickHandler={createBookCopyHandler} />
```

Comment:
I would suggest not to use upper case here, if you really need it you can set it with css